x-pack/test/security_solution_cypress/cypress/e2e/detection_response/prebuilt_rules/prebuilt_rules_install_update_workflows.cy.ts

```diff
@@ -119,9 +121,17 @@ describe.skip(
             addElasticRulesButtonClick();
 
             cy.get(INSTALL_ALL_RULES_BUTTON).should('be.enabled').click();
-            cy.get(TOASTER)
-              .should('be.visible')
-              .should('have.text', `${numberOfRulesToInstall} rules installed successfully.`);
+            cy.wait('@installPrebuiltRules', {
+              timeout: 60000,
+            }).then(() => {
+              cy.get(TOASTER)
+                .should('be.visible')
+                .should(
+                  'have.text',
+                  // i18n uses en-US format for numbers, which uses a comma as a thousands separator
+                  `${numberOfRulesToInstall.toLocaleString('en-US')} rules installed successfully.`
```

Comment:
@jpdjere have you tracked also down when the number formatting in the toast has changed and why the tests didn't catch it? I guess it was done before we started running tests for all PRs.

Reply:
No, this is because this test installs rules from the real package. This test has been skipped until now, and only recently did the number of rules in the package went over 999. Up to that number of rules there's no formatting (no thousands separator).
This is a "new" issue in that sense, but the code should always have been like that. We just didn't realize it before the rules reached 1000.